commands/deploy.go

```diff
@@ -70,9 +70,18 @@ func (c *DeployCmd) Run() (err error) {
 			return fmt.Errorf("not a valid Node.js app: \n\n%s", errstr)
 		}
 	}
+	
+	
+	envCI, err := strconv.ParseBool(os.Getenv("CI"))
```

Comment:
I think we should make use of Kong's env variable features rather than writing our own code to handle it, as the example here does https://github.com/section/sectionctl/blob/8c894a8f722c81ee2f2232aee5ee8be756cb0e3f/sectionctl.go#L37. This will help us keep naming the same. I'm actually not even sure if we should use a environment variable for this 'quiet' feature, as in a CI tool someone can just set the flag on the commands. `CI` is a very generic environment variable name, what if someone is using that in their own system and we unintentionally pick it up? What do you think about just going with the flag and no env variable?

Reply:
What if we set SECTION_CI env var?